containers/heavy_forwarder/bin/run.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
```

Comment:
Straight copy of https://github.com/openshift/splunk-forwarder-operator/blob/c28e0fb044330d413e18cd7f0a43ab1673d2aa18/containers/heavy_forwarder/bin/run.sh